LondonBritishLibrary/orient/BLorient13271.xml

```diff
@@ -0,0 +1,146 @@
+                                ጸሎታ፡ ወበረካታ፡ ወሀብተ፡ ረድኤታ፡ የሃሉ፡ ምስለ፡ ገብራ፡ <gap reason="lost"/> ዘጸለየት፡ ባቲ፡ አ<pb n="3v"/>መ፡ ፳ወ፩ለወርኀ፡ ሰኔ፡ በደብረ፡ ጎልጎታ፡ 
+                                ዘውእቱ፡ መቃብረ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="28v" to="38v"/>
+                            <title ref="LIT2444Temher"/>
+                            <incipit xml:lang="gez">በእንተ፡ ትምህርተ፡ ኅቡአት፡ ቅድመ፡ ዘትትነገር፡ እምጵርስፎራ፡ ለምዕመናን፡ ህቡአት፡ ትምህርት፡</incipit>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">42</measure>
+                                    <measure unit="leaf" type="blank" corresp="#1 #2 #39 #42v">4</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>75</height>
+                                        <width>55</width>
+                                    </dimensions>
+                                </extent></supportDesc>
+                            <layoutDesc>
+                                <layout writtenLines="10 13"/>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Poor handwriting.</desc>
+                                <date notBefore="1800" notAfter="1899" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="40r" to="42v"/>
+                                    <desc type="GuestText">A new beginning has been made after inverting the manuscript. A short calendar for the
+                                        commemoration of saints.</desc>
+                                    <q xml:lang="gez">ሠረቀ፡ መስከረም፡ አመ፡ ፩ዮሐንስ፡ ወበርተሎሜዎስ፡ ሠረቀ፡ ጥቅምት፡ አመ፡ ፲ወ፪ማቴዎስ፡ 
+                                        ሐዋ<supplied reason="undefined" resp="PRS8999Strelcyn">ርያ፡</supplied> አመ፡ ፳ወ፪ሉቃስ፡ <gap reason="ellipsis"/></q>
+                                </item>
+                                <item xml:id="e1">
+                                    <desc type="OwnershipNote">The name of the owner was 
+                                        <persName ref="PRS14342GabraMuse">Gabra Muse</persName>, 
```

Comment:
role="owner"
role="scribe"

please always assign roles to the persons relevant for the manuscript production

Reply:
Yes. These two escaped my attention.